jekyll/_docs/installing-elasticsearch.md

```diff
@@ -4,41 +4,41 @@ title: Install a custom version of Elasticsearch
 description: Installing a custom version of Elasticsearch
 ---
 
-To enable Elasticsearch, add the following to your circle.yml:
+To enable Elasticsearch, add the following to your `circle.yml`:
 
 ```
 machine:
   services:
     - elasticsearch
 ```
 
-The default version of elasticsearch is {{ site.data.precise.versions.elasticsearch }}.
-If you need a custom version, you can download and start it from your build. To install 1.0.1, add the following to your circle.yml:
+The default version of Elasticsearch is {{ site.data.precise.versions.elasticsearch }}.
```

Comment:
I would change to trusty, and maybe add a tad of text stating thats the image you are talking about. Remember we don't care about 12.04 anymore :)